src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -79,6 +79,7 @@ export const NavigationTabs: React.FC<NavigationTabsProps> = ({ partner }) => {
 
   return (
     <StickyContainer ContainerComponent={Container}>
+      <span id="jumpto-PartnerNavBar" />
       <FullBleed py={2}>
```

Comment:
Should be able to just pass that `id` to the `FullBleed` component

Reply:
I think I did it because ts swore at an unknown argument ID or smth like :)

Reply:
Ah, you're right; we need to extend the `React.HTMLAttributes<HTMLDivElement>` typing for `FullBleed`, I think. Is fine either way.

Reply:
fixed